src/main/java/core/basesyntax/MainApp.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import core.basesyntax.processing.CsvUtils;
+
+public class MainApp {
+    public static void main(String[] args) {
+        String csvInputFile = "src/main/resources/inputFile.csv";
```

Comment:
I do not understand necessary to use csv if you get your own file to aggregate. You can just use split by ",", I mean

Reply:
that is one of the recommendation from task description, and point here is to practice with usage of external libraries